data/com.endlessm.GoogleChromeInitialSetup.gschema.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If you remove this, it's unclear to me how you can still check if the schema exists and the value of its "legacy" setting. 
Since both the chrome helper and the schema are part of OSTree, and the schema is required to read the setting, there is no way for this chrome helper to perform the migration if you remove the schema.